templates/components/itilobject/service_levels.html.twig

```diff
@@ -162,7 +161,8 @@
                   </div>
                {% endif %}
 
-               {% if la_displayed %}
+               {% if (item.fields[la_field.lafieldname] != 0) %}
```

Comment:
I think the orignal check must be kept here because the field can be hidden by the template.